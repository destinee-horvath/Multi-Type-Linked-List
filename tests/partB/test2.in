NEW 0
INSERT 0 -1 element1
INSERT 0 -2 element2
INSERT 0 -3 element3
INSERT 0 -4 element4
INSERT 0 0 element5
NEW 3
a
d
end
INSERT 1 1 b
INSERT 1 2 c
INSERT 1 4 e
INSERT 1 0 begin
