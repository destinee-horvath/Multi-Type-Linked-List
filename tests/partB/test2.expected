List 0: 
List 0: element1
List 0: element2 -> element1
List 0: element3 -> element2 -> element1
List 0: element4 -> element3 -> element2 -> element1
List 0: element5 -> element4 -> element3 -> element2 -> element1
List 1: a -> d -> end
List 1: a -> b -> d -> end
List 1: a -> b -> c -> d -> end
List 1: a -> b -> c -> d -> e -> end
List 1: begin -> a -> b -> c -> d -> e -> end
